learningresources/tests/test_api.py

```diff
@@ -55,6 +57,11 @@ def test_create(self):
             create_course(**self.kwargs)
         self.assertTrue(course_count() == after)
 
+        self.assertRaises(
+            IntegrityError,
+            lambda: create_course(None, self.repo.id, 4, 9, self.user.id)
+        )
```

Comment:
This is actually a NOT NULL constraint error. I added a comment to make it clearer